### PR TITLE
OSDOCS-8834: Updated images to change format from SVG to PNG

### PR DESCRIPTION
**Pull request by EricPonvelle**
Version(s):
`enterprise-4.14+`

Issue:
[OSDOCS-8834](https://issues.redhat.com/browse/OSDOCS-8834)

Link to docs preview:
* AWS Architecture
    * [Before 1](https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/introduction_to_rosa/rosa-architecture#osd-aws-privatelink-architecture.adoc_rosa-architecture-models)
        image
    * [After 1](https://file.rdu.redhat.com/eponvell/OSDOCS-8834_Image-Updates/rosa_architecture/rosa_architecture_sub/rosa-architecture-models.html#rosa-architecture_rosa-architecture-models)
        image
    * [Before 2](https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/introduction_to_rosa/rosa-architecture#rosa-architecture_rosa-architecture-models)
        image
    * [After 2](https://file.rdu.redhat.com/eponvell/OSDOCS-8834_Image-Updates/rosa_architecture/rosa_architecture_sub/rosa-architecture-models.html#osd-aws-privatelink-architecture.adoc_rosa-architecture-models)
        image

* AWS Architecture for ROSA with HCP clusters
    * [Before](https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/install_rosa_with_hcp_clusters/rosa-hcp-sts-creating-a-cluster-quickly#rosa-hcp-classic-comparison_rosa-hcp-sts-creating-a-cluster-quickly)
        image
    * [After](https://file.rdu.redhat.com/eponvell/OSDOCS-8834_Image-Updates/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.html#rosa-hcp-classic-comparison-networks_rosa-hcp-sts-creating-a-cluster-quickly)
        image

* Remote Monitoring
    * [Before](https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/support/remote-health-monitoring-with-connected-clusters#understanding-telemetry-and-insights-operator-data-flow_about-remote-health-monitoring)
        image
    * [After](https://file.rdu.redhat.com/eponvell/OSDOCS-8834_Image-Updates/support/remote_health_monitoring/about-remote-health-monitoring.html#understanding-telemetry-and-insights-operator-data-flow_about-remote-health-monitoring)
        image
    
QE review:
- [ ] QE has approved this change.

Additional information:
The graphics team updated the images in our docs, converting these images from SVG to PNG. The SVG images caused issues within Customer Portal.

### CODE REVIEW

**Review by lpettyjo — COMMENTED** [review]
LGTM!

File: terraform.tfstate
Comment (jeana-redhat) [inline terraform.tfstate:1]: Is this in here on purpose?

File: modules/osd-aws-privatelink-architecture.adoc [hunk 1]
```diff
@@ -16,7 +16,7 @@ The following diagram shows network connectivity of a PrivateLink cluster.
 
 .Multi-AZ AWS PrivateLink cluster deployed on private subnets
 
-image::156_OpenShift_ROSA_Arch_0621_privatelink.svg[Multi-AZ AWS PrivateLink cluster deployed on private subnets]
+image::156_OpenShift_ROSA_Arch_1221_privatelink.png[Multi-AZ AWS PrivateLink cluster deployed on private subnets]
```
Comment (jeana-redhat) [inline modules/osd-aws-privatelink-architecture.adoc:19]: Just confirming: is the change from 0621 to 1221 intentional?

File: images/156_OpenShift_ROSA_Arch_0621_private_public_classic.png
Comment (jeana-redhat) [inline images/156_OpenShift_ROSA_Arch_0621_private_public_classic.png:1]: This one didn't have an SVG I guess?

**Review by jeana-redhat — COMMENTED** [review]
Hey, I was a bit confused about the 0621 vs. 1221 thing going on, especially since the first PNG here seems to not have an SVG version. Also not sure what the terraform files is doing/if it belongs.

LMK if you're sure about those things, and I can merge :)

/remove-label merge-review-in-progress
/remove-label merge-review-needed

File: modules/osd-aws-privatelink-architecture.adoc (hunk 1 above)
Comment (EricPonvelle) [reply modules/osd-aws-privatelink-architecture.adoc:19]: Yup, they should still be the same image.

File: images/156_OpenShift_ROSA_Arch_0621_private_public_classic.png
Comment (EricPonvelle) [reply images/156_OpenShift_ROSA_Arch_0621_private_public_classic.png:1]: I did delete the .SVGs, but I think there was some weird similar naming, so I renamed this one

File: terraform.tfstate
Comment (EricPonvelle) [reply terraform.tfstate:1]: Yup - good catch. That wasn't supposed to be there.

**Pull request merged**